libs/dh/globalization/assets-localization/src/assets/i18n/en.json

```diff
@@ -1562,7 +1562,9 @@
         "loading": "Creating request...",
         "success": "Your request has been created.",
         "error": "{{shared.error.title}}"
-      }
+      },
+      "createdMessage": "Your request has been created, and the hole log will be available through the B2B channel shortly.",
```

Comment:
question: should it be called "missing measurements log"?

Reply:
Well spotted, should be fixed now :)